app/mashup/Preload.scala

```diff
@@ -142,6 +156,13 @@ final class Preload(
         }
     }
 
+  private def teams(me: Me): Fu[List[lila.team.TeamId]] =
+    teamCache
+      .teamIdsList(me)
+      .flatMap:
+        _.filterA:
+          teamCache.forumAccess.get(_).map(_ != lila.core.team.Access.None)
```

Comment:
`teamCache.forumAccess.get` will be called on every homepage load, as many times as the player has teams joined. That's up to 50. We better keep an eye on [monitoring](https://monitor.lichess.ovh/d/iFvd7urZk/cache-caffeine-per-cache?orgId=1&var-instance=lila@ocean&var-field=mean&var-name=team.forum.access)